### ISSUE TITLE
Windows Defender flagging pscp.exe as a virus now...

### ISSUE THREAD

**Issue by gtwy** [lalbornoz/PuTTie#5]
It's quarantining it for this reason. https://www.microsoft.com/en-us/wdsi/threats/malware-encyclopedia-description?Name=Trojan:Script/Wacatac.H!ml

Not sure what it means. Is it a false positive?

**Comment by lalbornoz** [comment]
Yeah, that's a false positive. The "ml" stands for machine learning I think?

**Comment by SanderGit** [comment]
pageant.exe is now flagged with Wacatac.B!ml 

I can no longer run this on my work laptop. Could you please [submit your files beforehand](https://www.microsoft.com/en-us/wdsi/filesubmission/) to avoid this from happening in the future?

**Comment by lalbornoz** [comment]
> pageant.exe is now flagged with Wacatac.B!ml
> 
> I can no longer run this on my work laptop. Could you please [submit your files beforehand](https://www.microsoft.com/en-us/wdsi/filesubmission/) to avoid this from happening in the future?

I've submitted both the file- and the Registry-based pageant.exe and putty.exe releases.

> Additionally pageant.exe seems to be contacting an IP address with reputation issues:
> 
> https://www.virustotal.com/gui/file/98b1126c6cbf6b007afedc35bf30bebe72e557d7af419bec2a7b8aafe15715d1/behavior https://www.virustotal.com/gui/ip-address/23.216.147.64

That's definitely incorrect, as is much of the other information on that site concerning pageant.exe, e.g. "Keylogging T1056.001 log keystrokes via polling", etc. Neither the PuTTY upstream nor my own code in Pageant contacts any such IP address or even any host at all except for those explicitly specified by the user.

**Comment by gtwy** [comment]
Note that the newest putty.exe that fixes the crashing issue isn't in the latest release yet

**Comment by lalbornoz** [comment]
It is (or should be) actually:

https://github.com/lalbornoz/PuTTie/releases/tag/PuTTie-file-Release-1bfd0b26
https://github.com/lalbornoz/PuTTie/releases/tag/PuTTie-registry-Release-1bfd0b26
